src/commands/exec.yml

```diff
@@ -0,0 +1,18 @@
+description: Run a command with secret environment variables loaded from SecretHub
+parameters:
+  command:
+    type: string
+    description: Command to execute with secrets
+  version:
+    type: string
+    default: latest
+    description: Version of the SecretHub CLI
+  flags:
+    type: string
+    default: ''
+    description: Flags to pass to the `secrethub run` command
```

Comment:
I believe `--no-prompt` should be enabled by default.

Reply:
By default or always?

Reply:
Always :+1:

Reply:
We could make it configurable in it's own parameter if we find a use case.